src/PKSim.Core/Repositories/IIndividualParametersSameFormulaOrValueForAllSpeciesRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Utility.Collections;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Repositories
+{
+   public interface IIndividualParametersSameFormulaOrValueForAllSpeciesRepository : IStartableRepository<IndividualParameterSameFormulaOrValueForAllSpecies>
+   {
+      bool IsSameFormula(ParameterMetaData parameterMetaData);
+      bool IsSameValue(ParameterMetaData parameterMetaData);
+      (bool IsSameFormula, bool IsSameValue) IsSameFormulaOrValue(ParameterMetaData parameterMetaData);
+   }
```

Comment:
@msevestre I was not quite sure which querys for the  repo are required, so I defined this 3 functions first (with `ParameterMetaData `as an argument).
if we need e.g. to query by {ContainerId, ParameterName} etc. - we can extend the interface.